<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plot_bar_widget</class>
 <widget class="QWidget" name="plot_bar_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="bar_label_count">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Count Variable:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bar_cb_count"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="bar_label_families">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Families:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bar_cb_families"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="bar_box_percentage">
       <property name="text">
        <string>As Percentage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bar_box_stacked">
       <property name="text">
        <string>Stacked Bars</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
